torch/tensor.py

```diff
@@ -371,6 +371,24 @@ def __invert__(self):
     def __hash__(self):
         return id(self)
 
+    def __int__(self):
+        if self.numel() == 1:
+            return int(self.storage()[0])
+        raise TypeError("only 1-element tensors can be converted "
+                        "to Python scalars")
```

Comment:
should we be explicit about the type?  "Python scalars" aren't actually a defined type, right?

Reply:
I was inspired by `numpy`'s error message:
```
TypeError: only length-1 arrays can be converted to Python scalars
```
Of course it won't harm for `torch` to be more verbose